[Service Fabric] adding api version 2021-06-01

a-santamaria opened the pull request:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [x] PowerShell
      - [x] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [x] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

Commit:
initiall commit with previous stable version 2020-03-01

Commit:
update to version 2021-06-01

Commit:
update examples with 2021-06-01 api

Commit:
fix automation errors

Commit:
remove operation definitions from app

chiragg4u commented on specification/servicefabric/resource-manager/Microsoft.ServiceFabric/stable/2021-06-01/application.json:
```diff
@@ -0,0 +1,2285 @@
+      ]
+    }
+  ],
+  "paths": {
```
>paths [](http://example.com/codeflow?start=3&length=5)

Are you sure you want to remove operations API?

a-santamaria replied:
we already have the operation api in clusters.json. do we need to keep it in both?

chiragg4u commented on specification/servicefabric/resource-manager/Microsoft.ServiceFabric/stable/2021-06-01/application.json:
```diff
@@ -0,0 +1,2285 @@
+      },
+      "description": "The error details."
+    },
+    "SystemData": {
```
>SystemData [](http://example.com/codeflow?start=5&length=10)

Please refer common definition from [https://github.com/Azure/azure-rest-api-specs/blob/bf44a3b7e5817ffc57b205a6d63c283af96d222b/specification/common-types/resource-management/v2/types.json](https://github.com/Azure/azure-rest-api-specs/blob/bf44a3b7e5817ffc57b205a6d63c283af96d222b/specification/common-types/resource-management/v2/types.json)

a-santamaria replied:
@chiragg4u when I refer to the common type I start getting these errors: https://github.com/Azure/azure-rest-api-specs/pull/14838/checks?check_run_id=2850749457. Am I doing something wrong?

chiragg4u commented on specification/servicefabric/resource-manager/Microsoft.ServiceFabric/stable/2021-06-01/cluster.json:
```diff
@@ -0,0 +1,1714 @@
+        "$ref": "#/definitions/ServiceTypeHealthPolicy"
+      }
+    },
+    "SystemData": {
```
>SystemData [](http://example.com/codeflow?start=5&length=10)

Same comment as in other file. Please refer common definitions
[https://github.com/Azure/azure-rest-api-specs/blob/bf44a3b7e5817ffc57b205a6d63c283af96d222b/specification/common-types/resource-management/v2/types.json](https://github.com/Azure/azure-rest-api-specs/blob/bf44a3b7e5817ffc57b205a6d63c283af96d222b/specification/common-types/resource-management/v2/types.json)

chiragg4u commented:
Thank you for copying the existing version into new folder for first commit. It helps in reviewing the change. Changes looks good from ARM perspective, signedoff, please follow the process given @ https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review to continue

Commit:
use systemData from common-types

Commit:
add type

Commit:
update vmInstanceCount description and min

Commit:
adding missing az params

leni-msft commented:
@a-santamaria please follow the Action in above comment to request cross-version breaking changes approval for `1011 - AddingResponseCode`

Commit:
adding missing cluster parameters

Commit:
remove 200 from app delete operations

Commit:
remove default respopnse from clusterVersions

a-santamaria commented:
> @a-santamaria please follow the Action in above comment to request cross-version breaking changes approval for `1011 - AddingResponseCode`

@leni-msft I removed those responses added. I added them by mistake.
